### PR TITLE
feat(hapi): add category on income transaction

### LINKED ISSUE #82: Create categories "claimed" a "unclaimed"
Create categories "claimed" and "unclaimed" on "income" types transaction

### PR DESCRIPTION
**Pull request by JAMares**
### What does this PR do?

- Resolve #82 

### Steps to test

1. Run project
2. Open localhost:9695
3. See on eden_transaction category "unclaimed" on income type transaction

#### CheckList

- [X] Follow proper Markdown format
- [X] The content is adequate
- [X] The content is available in both english and spanish
- [X] I Ran a spell check

### CODE REVIEW

File: hapi/src/services/eden-historic-delegates.service.js [hunk 1]
```diff
@@ -89,6 +95,65 @@ const registerHistoricDelegate = async (date, account, rank) => {
...
+    )
+    LASTEST_RATE_DATA_CONSULTED = data.market_data.current_price.usd
+    LASTEST_RATE_DATE_CONSULTED = txDate
+  }
+
+  try {
+    const election = await edenHistoricElectionGql.get({
+      date_election: { _lte: date }
+    })
+
```
Comment (leisterfrancisco) [inline hapi/src/services/eden-historic-delegates.service.js:125]: remove empty space

File: hapi/src/services/eden-historic-delegates.service.js [hunk 2]
```diff
@@ -89,6 +95,65 @@ const registerHistoricDelegate = async (date, account, rank) => {
...
+    const delegateData = await edenDelegatesGql.get({
+      account: { _eq: account }
+    })
+
```
Comment (leisterfrancisco) [inline hapi/src/services/eden-historic-delegates.service.js:129]: remove empty space

File: hapi/src/services/eden-historic-delegates.service.js [hunk 3]
```diff
@@ -89,6 +95,65 @@ const registerHistoricDelegate = async (date, account, rank) => {
...
+    const electionData = await edenElectionGql.get({
+      id_delegate: { _eq: delegateData.id },
+      election: { _eq: election.election }
+    })
+
```
Comment (leisterfrancisco) [inline hapi/src/services/eden-historic-delegates.service.js:134]: remove empty space

File: hapi/src/services/eden-historic-delegates.service.js [hunk 4]
```diff
@@ -1,11 +1,17 @@
...
+  edenElectionGql,
+  edenTransactionGql
 } = require('../gql')
 const { servicesConstant } = require('../constants')
 const { sleepUtil, eosUtil, axiosUtil } = require('../utils')
-const { hyperionConfig, edenConfig } = require('../config')
+const { hyperionConfig, edenConfig, eosConfig } = require('../config')
+
+let LASTEST_RATE_DATE_CONSULTED = null
+let LASTEST_RATE_DATA_CONSULTED = null
```
Comment (leisterfrancisco) [inline hapi/src/services/eden-historic-delegates.service.js:14]: Why are these variables declared here?

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

File: hapi/src/services/eden-historic-delegates.service.js (hunk 4 above)
Comment (JAMares) [reply hapi/src/services/eden-historic-delegates.service.js:14]: To avoid consulting the same date several times

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**